modules/router-store/project.json

```diff
@@ -43,12 +43,8 @@
       "outputs": ["{options.outputFile}"]
     },
     "test": {
-      "executor": "@nx/jest:jest",
-      "options": {
-        "jestConfig": "modules/router-store/jest.config.ts",
-        "runInBand": true,
-        "passWithNoTests": false
-      },
+      "executor": "@analogjs/vitest-angular:test",
```

Comment:
Could you also include some additional commands for the build-step please, similar to 

https://github.com/ngrx/platform/blob/main/modules/store/project.json#L28-L36